internal/registry/registry_cache.go

```diff
@@ -47,6 +47,11 @@ type Entry struct {
 	Buildpacks []Buildpack `json:"buildpacks"`
 }
 
+// ExtensionEntry is a list of extensions stored in a registry
+type ExtensionEntry struct {
+	Extensions []Buildpack `json:"extensions"`
+}
+
```

Comment:
We might be able to reduce some duplication by renaming `BuildpackEntry.Buildpacks` to `ModuleEntry.Modules` and re-using the same struct